DSESwift/DataStorageEngine.swift

```diff
@@ -31,12 +30,12 @@ public class DataStorageEngine{
     // reference to the datasyncer
     var dataSyncer = DataSyncer()
     
-    private var dataSyncerProgressHandler = DataSyncerProgressHandler()
+    private var dataSyncerCallbackHandler = DataSyncerCallbackHandler()
 
     
     //This prevents others from using the default '()' initializer for this class.
-    private init() {
-        self.dataSyncer.delegate = dataSyncerProgressHandler
+    private override init() {
+        self.dataSyncer.delegate = dataSyncerCallbackHandler
     }
     
```

Comment:
@kanekin is getInstance safe? will it always return actual and initialized instance?